OVS_Bonding/RawFiles/setup-ovs.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+set -ex
+
+if [[ $(nmcli conn | grep -c ovs) -eq 0 ]]; then
+  echo "configure ovs bonding"
+  primary_mac=$(cat /boot/mac_addresses | awk -F= '/PRIMARY_MAC/ {print $2}')
```

Comment:
I'm not sure I understand the process that copies  `/var/mnt/mac_addresses` to `/boot/mac_addresses`. Can you elaborate?

Moreover, theoretically can this mechanism be used to create a log file during ignition process that will later be copied to /boot as well? (I'm asking because I tried and it didn't work..)

Reply:
In the case that you want to transfer something from the live Ignition and the OS , It will be saved on /boot .
In order to make it clearer I changed the mount point to: 
Mount : /boot 
File save location : /boot/mac_addresses 
*no need to unmount it will do it automatically but in case you want to , you can do it yourself